.github/workflows/build.yml

```diff
@@ -32,7 +32,8 @@ jobs:
           files: core/build/reports/jacoco/testCodeCoverageReport/testCodeCoverageReport.xml,cli/base/build/reports/jacoco/testCodeCoverageReport/testCodeCoverageReport.xml,cli/lfc/build/reports/jacoco/testCodeCoverageReport/testCodeCoverageReport.xml,cli/lfd/build/reports/jacoco/testCodeCoverageReport/testCodeCoverageReport.xml,cli/lff/build/reports/jacoco/testCodeCoverageReport/testCodeCoverageReport.xml
 
   epoch:
-    uses: lf-lang/epoch/.github/workflows/build.yml@main
+    uses: lf-lang/epoch/.github/workflows/build.yml@build-against-fork
```

Comment:
Change back to `main` after https://github.com/lf-lang/epoch/pull/53 has been merged.